agent/engine/docker_task_engine_test.go

```diff
@@ -2821,3 +2870,77 @@ func TestCreateFirelensContainerSetFluentdUID(t *testing.T) {
 	ret := taskEngine.(*DockerTaskEngine).createContainer(testTask, testTask.Containers[0])
 	assert.NoError(t, ret.Error)
 }
+
+func isEnvPresent(expected string, actual []string) bool {
```

Comment:
i think you can just use assert.Contains instead of this

Reply:
Didn't work. I tried it